projects/plugins/jetpack/modules/shortcodes/vimeo.php

```diff
@@ -250,6 +259,7 @@ class_exists( 'Jetpack_AMP_Support' )
  * Callback to modify output of embedded Vimeo video using Jetpack's shortcode.
  *
  * @since 3.9
+ * @deprecated since x.x.x
```

Comment:
You can use the `$$next-version$$` placeholder; it will be automatically replaced on release.

https://github.com/Automattic/jetpack/blob/d3f58235992e84fcc8a4b1bc191c56ba6edf213a/projects/packages/README.md#L116-L123

```suggestion
 * @deprecated since $$next-version$$
```